ui-modules/blueprint-composer/app/components/dsl-editor/dsl-editor.js

```diff
@@ -92,6 +92,8 @@ export function dslEditorDirective($rootScope, $filter, $log, brUtilsGeneral, bl
             let name = item.name;
             if (attrs.length > 0) {
                 name += ` (${attrs.join(', ')})`;
+            } else {
```

Comment:
Might be cleaner as

```
let { name, entity, id } = item;
name += (attrs.length > 0)
  ? ` (${attrs.join(', ')})`
  : ` (${entity.id || id})`;
```

Reply:
Addendum: if this is used, remember to adjust the return statement as well.
```
    return {
        id,
        label: name,
        scope: 'On specific entity',
    };
```

Reply:
Good suggestion, I'll take it if more changes will be required in this PR.

Reply:
Improved in a6899562a7a127dac99f64040f90f8441bc7d305.